replacing-databases-bigquery/README.md

```diff
@@ -0,0 +1,32 @@
+# Replicating databases to BigQuery
+
+This is a stack that deploys a Three-tier Autoscaling Web Application using GKE.
+
+This repository contains Terraform sample code to deploy Gke, Cloud Storage and Cloud SQL private instances, and establish communication between them.
+
+Resources created
+- VPC
+- Data Stream
+- Cloud SQL for MySQL
+- Cloud Storage
+
+![arquitecture](architecture.png)
+
+## Deploy
+
+1. Click on Open in Google Cloud Shell button below.
+<a href="https://ssh.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/click-to-deploy-solutions&cloudshell_workspace=replacing-databases-bigquery" target="_new">
```

Comment:
Please review the directory name. I think you meant `replicating` instead of `replacing`